File: FindNow_final/entidades/src/main/java/com/example/entidades/Asignacion.java
package com.example.entidades;

/**
 * Created by XL on 3/1/2017.
 */

public class Asignacion {
    public Asignacion() {
    }
    private int id_asignacion;
    private Usuario usuario ;
    private Dispositivo dispositivo ;
    private int propietario ;

    public int getId_asignacion() {
        return id_asignacion;
    }

    public void setId_asignacion(int id_asignacion) {
        this.id_asignacion = id_asignacion;
    }

    public Usuario getUsuario() {
        return usuario;
    }

    public void setUsuario(Usuario usuario) {
        this.usuario = usuario;
    }

    public Dispositivo getDispositivo() {
        return dispositivo;
    }

    public void setDispositivo(Dispositivo dispositivo) {
        this.dispositivo = dispositivo;
    }

    public int getPropietario() {
        return propietario;
    }

    public void setPropietario(int propietario) {
        this.propietario = propietario;
    }
}

File: findMe Backend/CapaEntidades/Asignacion.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CapaEntidades
{
  public  class Asignacion
    {
         public int id_asignacion { get; set; }
        public Usuario usuario_ { get; set; }
        public Dispositivo dispositivo { get; set; }
        public int propietario { get; set; }

        public Asignacion()
        {
            usuario_ = new Usuario();
            dispositivo = new Dispositivo();
        }
    }
}
